obspy/signal/spectral_estimation.py

```diff
@@ -1737,6 +1821,13 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 selected.
         :type show_noise_models: bool, optional
         :param show_noise_models: Enable/disable plotting of noise models.
+        :type show_earthquakes: bool, optional
+        :param show_earthquakes: Enable/disable plotting of earthquake models like
+                in [ClintonHeaton2002]_ and [CauzziClinton2013]_. Defaults to 
+                ``False``.
+        :type eqlims: tuple of 4 floats, optional
+        :param eqlims: Ranges (minimum and maximum) for magnitude and distance of 
+                earthquake models. Defaults of ``(0, 5, 0, 99)``.
```

Comment:
We already have tons of kwargs in the plot method. Please compress these two kwargs into a single one, I don't think we lose anything doing that.

```
        :type show_earthquakes: None or tuple of 4 floats
        :param show_earthquakes: Enable/disable plotting of earthquake models like
                in [ClintonHeaton2002]_ and [CauzziClinton2013]_. Disabled by default
                (``None``). Specify ranges (minimum and maximum) for magnitude and distance of 
                earthquake models given as four floats, e.g. ``(0, 5, 0, 99)`` for magnitude 0-5
                at a epicentral distance range of 0-99 km.
```

in code..
```python
        if show_earthquakes is not None:
            min_mag, max_mag, min_dist, max_dist= show_earthquakes
            ...
```

That'll also improve the readability of that if clause further down.